src/http/server_context.rs

```diff
@@ -9,7 +10,7 @@ use crate::http::HttpDataLoader;
 use crate::lambda::{Expression, Unsafe};
 
 pub struct ServerContext {
-  pub schema: dynamic::Schema,
+  pub schema: RwLock<dynamic::Schema>,
```

Comment:
Try to write code that you can reason about by looking at locally. For eg: Adding internal mutability of a particular type, and then not using it anywhere within the server context can be confusing. One way to fix this is by moving the polling/reloading logic into server context, and making all fields private. I think this is also a problem with http_client.